crates/backup-cli/src/types.rs

```diff
@@ -1,7 +1,9 @@
 use ed25519_dalek::SigningKey;
+use mpc_node::keyshare::Keyshare;
 use rand_core::CryptoRngCore;
 use serde::{Deserialize, Serialize};
-pub struct KeyShares {}
+
+pub struct KeyShares(pub Vec<Keyshare>);
```

Comment:
Adding this instead of using simply `Vec<Keyshare>` made the code slightly worse in a few places, let me know if it is ok not to use this type

Reply:
It is quite simple indeed, but still would make it different from the rest of the implementation, and does not bring any gain at least right now. Anyhow, it should be very easy to put back in the future if we actually need something. The definition of this type will probably be defined only once in the node-types crate. At that moment it might be worth it to define a proper type.

Reply:
Fair enough.